File: Python/List - 2/sum67.py
def sum67(nums):
    
    total = 0
    
    omit = False
    
    for i in nums:
        
        if i == 6:
            
            omit = True
            
            continue
        
        elif i!=7 and omit:
            
            continue
        
        elif i == 7 and omit:
            
            omit = False
            
            continue
        
        else:
            
            total += i
            
    return total

File: Java/Array - 2/sum67.java
public int sum67(int[] nums) {

    int sum=0;
  
    for(int i=0;i<nums.length;i++) {
    
        if(nums[i]==6)
        
        while(nums[i]!=7 ) {
        
            i++;
        }
    
        else
            
        sum += nums[i];
        
    }
    
    return sum;
    
}